zebra-chain/src/transaction/tests.rs

```diff
@@ -1,7 +1,121 @@
-use crate::serialization::{ZcashDeserialize, ZcashSerialize};
+use proptest::{
+    arbitrary::{any, Arbitrary},
+    collection::vec,
+    option,
+    prelude::*,
+};
+
+use crate::{
+    serialization::{ZcashDeserialize, ZcashSerialize},
+    types::LockTime,
+};
 
 use super::*;
 
+#[cfg(test)]
+impl Transaction {
+    pub fn v1_strategy() -> impl Strategy<Value = Self> {
```

Comment:
Should these be `pub(crate)`? Or is that not important because they're `#[cfg(test)]`?

Reply:
I think until they are needed beyond the crate, I'll keep them private and behind the test flag.